hypothesis-python/tests/cover/test_searchstrategy.py

```diff
@@ -93,3 +95,9 @@ def test_can_map_nameless():
 def test_can_flatmap_nameless():
     f = nameless_const(just(3))
     assert repr(f) in repr(integers().flatmap(f))
+
+
+def test_flatmap_with_invalid_expand():
```

Comment:
A bit unsure if this is the proper way to do things, please advise if not.

Reply:
I think replacing the `do_draw` with a simple `.example()` should do the trick here.